arches/app/templates/views/components/widgets/resource-instance-select.htm

```diff
@@ -219,26 +219,26 @@
                 </div>
             </div>
             <!-- /ko -->
-    
+
             <!-- ko if: relationshipsInFilter().length > 0-->
             <div class="rr-table">
                 <div data-bind='foreach: relationshipsInFilter' style="display: flex; flex-direction: column;">
                     <div class='rr-table-row'>
                         <div class="rr-table-row-initial">
                             <div class='rr-table-column icon-column'>
                                 <button
-                                    data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false">
+                                    data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false" aria-label="Edit Resource Instance">
                                     <i class="fa fa-pencil"></i>
                                 </button>
                             </div>
                             <div class='rr-table-column icon-column'>
-                                <button data-bind="click: $parent.deleteRelationship, clickBubble: false">
+                                <button data-bind="click: $parent.deleteRelationship, clickBubble: false"  aria-label="Delete Resource Instance">
```

Comment:
another "delete resource" label that should say something like "remove resource instance relationship"

Reply:
Label modified